doc/changelog.rst

```diff
@@ -1,6 +1,61 @@
 Changelog
 =========
 
+0.3.0
+-----
+
+Expected Release on Febuary 11, 2022
+
+Description
+
+ - Update Redis version to 1.3.2 (PR162_)
+ - Streamline ``run`` and ``connect`` commands for Redis and RedisCluster (PR203_)
+ - Update handling of command replies (PR138_) (PR163_)
+ - Add SmartRedis namespace to ``srassert.h`` (PR191_)
+ - Expose Redis commands to the Python client (PR139_) (PR160_)
+ - Extend inverse CRC16 prefixing to all hash slots (PR161_)
+ - Refactor the dataset acknowledgement key for easier communication with ensambles (PR195_)
+ - Add ``poll_dataset`` functionality to all SmartRedis clients (PR184_)
+ - Add commands to retrieve statistics on scripts and models in Python and C++ clients (PR197_)
+ - Add environment variables to control client runtime behavior (PR194_)
+ - Create a Dockerfile for SmartRedis (PR180_)
+ - Simplify SmartRedis build proccess (PR189_)
+ - Impove error handling across all SmartRedis clients (PR159_) (PR199_) (PR205_) (PR206_)
```

Comment:
Maybe we move error handling, new commands (194, 197,184), and docker Dockerfile towards the top since users will be most interested in these.

Reply:
Makes sense! Moved in https://github.com/CrayLabs/SmartRedis/pull/209/commits/a8561c47fe1c2d31271f36ac01b711311ae7d028